index.d.cts

```diff
@@ -0,0 +1,2 @@
+export * from ".";
+export { default } from ".";
```

Comment:
What does this imply? Can we model the `module.exports = test` behavior, as well as exporting the types?

Reply:
Hey, I've gotten a bit better of an idea now as to how this new typing thing works. I'll update this pr soon to improve things.